extensions/wikia/WikiaMaps/tests/WikiaMapsPoiControllerTest.php

```diff
@@ -72,77 +72,10 @@ public function testDeletePoi_throws_permission_error_when_blocked() {
 			->with( 'poiId' )
 			->willReturn( 1 );
 
-		$controllerMock->wg->User = $userMock;
+		$controllerMock->getApp()->wg->User = $userMock;
 
 		$controllerMock->deletePoi();
 	}
-	
-	/**
-	 * @dataProvider isValidUrlDataProvider
-	 */
-	public function testIsValidUrl( $description, $urlMock, $isValidCalledTimes, $isValidMock, $expected ) {
-		$poiControllerMock = $this->getMockBuilder( 'WikiaMapsPoiController' )
-			->disableOriginalConstructor()
-			->setMethods( [ 'getData' ] )
-			->getMock();
-
-		$poiControllerMock->expects( $this->once() )
-			->method( 'getData' )
-			->with( 'articleTitleOrExternalUrl' )
-			->willReturn( $urlMock );
-
-		$wikiaValidatorUrlMock = $this->getMockBuilder( 'WikiaValidatorUrl' )
-			->disableOriginalConstructor()
-			->setMethods( [ 'isValid' ] )
-			->getMock();
-
-		$wikiaValidatorUrlMock->expects( $this->$isValidCalledTimes() )
-			->method( 'isValid' )
-			->willReturn( $isValidMock );
-
-		/** @var WikiaMapsPoiController $poiControllerMock */
-		$this->assertEquals( $poiControllerMock->isValidUrl( $wikiaValidatorUrlMock ), $expected, $description );
-	}
-
-	public function isValidUrlDataProvider() {
-		return [
-			[
-				'empty BUT valid URL',
-				'urlMock' => '',
-				'isValidCalledTimes' => 'never',
-				'isValidMock' => false,
-				'expected' => true,
-			],
-			[
-				'not empty, valid URL',
-				'urlMock' => 'http://www.wikia.com',
-				'isValidCalledTimes' => 'once',
-				'isValidMock' => true,
-				'expected' => true,
-			],
-			[
-				'not empty, valid URL - HTML5 URL encoded',
-				'urlMock' => 'http://www.wikia.com/%D0%A1%D0%BB%D1%83%D0%B6%D0%B5%D0%B1%D0%BD%D0%B0%D1%8F',
-				'isValidCalledTimes' => 'once',
-				'isValidMock' => true,
-				'expected' => true,
-			],
-			[
-				'not empty, valid URL with UTF8 characters',
-				'urlMock' => 'http://www.wikia.com/Служебная',
-				'isValidCalledTimes' => 'once',
-				'isValidMock' => true,
-				'expected' => true,
-			],
-			[
-				'not empty and invalid URL',
-				'urlMock' => 'This is not an URL',
-				'isValidCalledTimes' => 'once',
-				'isValidMock' => false,
-				'expected' => false,
-			]
-		];
-	}
```

Comment:
Yeah, this essentially tests `WikiaUrlValidator` class - which already has its own, separate test.